src/components/RangeSelector.js

```diff
@@ -92,6 +93,45 @@ class RangeSelector extends React.Component {
     });
   }
 
+  _handleTrackMouseDown (e) {
+    this.setState({
+      trackClicked: true
+    });
```

Comment:
Setting state this way will cause 2 state updates with the conditionals below. A way we've gotten around this is to put the updated state into a `const`, then only call `setState` once at the end of the method:
```javascript
_handleTrackMouseDown (e) {
  const updatedState = {
    trackClicked: true
  };

  // ...stuff

  if (newPixels < this.state.lowerPixels) {
    updatedState.dragging = 'Lower';
  }

  // ..other stuff

  this.setState(updatedState);
}
```